mat-process-api.Tests/V1/Validators/GetProcessRequestValidatorTest.cs

```diff
@@ -19,28 +19,45 @@ public void SetUp()
             validator = new GetProcessDocumentRequestValidator();
         }
 
-        [Test]
         [TestCase(null)]
         [TestCase("")]
         [TestCase(" ")]
-        public void given_null_or_empty_or_whitespace_processRef_the_validator_should_return_an_error(string processRef)
+        public void given_null_or_empty_or_whitespace_processRef_the_GetProcessDocumentRequestValidator_should_return_an_error(string processRef)
         {
             //assert
             validator.ShouldHaveValidationErrorFor(req => req.processRef, processRef);
         }
 
-        [Test]
-        public void given_invalid_guid_string_the_validator_should_return_an_error()
+        [TestCase(null)]
+        public void given_null_processRef_the_GetProcessDocumentRequestValidator_should_return_a_correct_error_message(string processRef)
         {
-            //arrange
-            string processRef = faker.Random.String();
+            //assert
+            var result = validator.ShouldHaveValidationErrorFor(req => req.processRef, processRef);
+            result.WithErrorMessage("Process reference must be provided.");
+        }
 
+        [TestCase("")]
+        [TestCase(" ")]
+        public void given_empty_or_whitespace_processRef_the_GetProcessDocumentRequestValidator_should_return_a_correct_error_message(string processRef)
+        {
+            //assert
+            var result = validator.ShouldHaveValidationErrorFor(req => req.processRef, processRef);
+            result.WithErrorMessage("Process reference cannot be empty.");
+        }
+
+        [TestCase("fe7ce7fb-7733-e877-a28c-f0857e6962b")] //less characters than it should contain
+        [TestCase("6418aa75-f5ee6975-5279d7bcfee5f948")] //missing dashes
+        [TestCase("a98e73-4abd0-83542-4b73225-72b6adc65")] //dashes in the wrong places
+        [TestCase("ju5t70m3-w0rd-th4t-w1ll-m34nn0th1ng0")] //uses characters that are not among hexadecimal numerals
+        [TestCase("randomstring")]
```

Comment:
Sorry, given you bad advice, now that I think a bit more, this 5th test case (`[TestCase("randomstring")]`) is not necessary because it falls under 2 categories tested above anyway. The categories are:
1) Less characters than it should contain
2) Uses characters not among Hex numerals
So it ends up testing things that were tested anyway. Better to take it out.

Reply:
I've removed that test case